[Term Entry] PyTorch Tensor Operations: .abs() (#6278)
Codecademy/docs: content/pytorch/concepts/tensor-operations/terms/abs/abs.md

---
Title: '.abs()'

> Description: 'Computes the absolute value of each element in a PyTorch tensor. For complex numbers, returns their magnitude.'

Reviewer: ```suggestion
Description: 'Computes the absolute value of each element in a PyTorch tensor, returning the magnitude for complex numbers.'
```

Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Deep Learning'
  - 'Programming'
  - 'PyTorch'
CatalogContent:
  - 'intro-to-py-torch-and-neural-networks'
  - 'paths/data-science'
---

> The **`.abs()`** method in PyTorch calculates the absolute value (magnitude) of each element in a tensor. For real numbers, this returns their non-negative value. For complex numbers, it computes the magnitude using the formula √(real² + imag²). This operation is widely used in data preprocessing, signal processing, and mathematical transformations.

Reviewer: ```suggestion
The **`.abs()`** method in PyTorch computes the absolute value of each tensor element. For real numbers, it returns the non-negative value. For complex numbers, it calculates the magnitude using √(real² + imag²). This function is useful in data preprocessing, signal processing, and mathematical transformations.
```

## Syntax

```pseudo

> tensor.abs()

Reviewer: Let us use the following syntax as per the documentation:

```pseudo
torch.abs(input)
```
- input (Tensor): The input tensor (can be real or complex).

refer to the documentation: https://pytorch.org/docs/stable/generated/torch.abs.html
